docs: add CONTRIBUTING guidelines (#528)
holdex/holdex-venture-studio: .github/CODE_OF_CONDUCT.md

# Code of Conduct

1. Constantly align yourself with our core [Holdex values](https://holdex.io/about).

> 2. Ensure the quality of your work. Unless you are a begginer, it is a sign of poor ownership and lack of confidence when you expect someone else to review your uncomplete work.

Reviewer: ```suggestion
2. Ensure the quality of your work. Unless you are a beginner, it is a sign of poor ownership and lack of confidence when you expect someone else to review your incomplete work.
```

> 3. Assign yourself to the Problem you are solving so others can are aware of it.

Reviewer: ```suggestion
3. Assign yourself to the Problem you are solving so others are aware of it.
```

> 4. Don’t use `@mention` when it is not crutial for the person to be notified. Use "reactions" (likes) instead.

Reviewer: ```suggestion
4. Don’t use `@mention` when it is not crucial for the person to be notified. Use "reactions" (likes) instead.
```
